tests/integration/conftest.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
Actually we should add ``fix-encoding-pragma`` hook to the codebase since we're supporting python2 and python3 at the same time.